Editor/CanvasView.cs

```diff
@@ -244,35 +245,55 @@ public void Load(Graph graph)
 
             // TODO: Move into reflection
             var attrs = graph.GetType().GetCustomAttributes(true);
+
+
             foreach (var attr in attrs)
             {
+                //Add Tags for search provider
                 if (attr is IncludeTagsAttribute include)
                 {
                     searchWindow.IncludeTags.AddRange(include.Tags);
                 }
+
+                //Add Required nodes from GraphAttributes
+                if (attr is RequireNodeAttribute required)
+                {
+                    if (searchWindow.IncludeTags.Contains(required.type.Namespace.ToString()))

```

Comment:
Remove this wrapping `if` condition - if the developer is adding a `[RequireNode]` attribute onto their graph, they intend to add that node. So we shouldn't add another condition here that may prevent that from happening.